lib/app_profiler.rb

```diff
@@ -162,6 +164,15 @@ def profile_sampler_enabled
       false
     end
 
+    def otel_instrumentation_enabled=(value)
+      require("opentelemetry/instrumentation/rack") if value
```

Comment:
```suggestion
      if value
        gem("opentelemetry-instrumentation-rack")
        require("opentelemetry/instrumentation/rack")
      end
```

Reply:
I am not sure if I understand the comment here 🤔 

>A log entry could easily be missed, approved, and deployed. We don't edit Rails config after boot, so this should be safe to do.

RE: `gem("opentelemetry-instrumentation-rack")` 👍 

>  If you will be activating the latest version of a gem, there is no need to call Kernel#gem, Kernel#require will do the right thing for you.

https://github.com/rubygems/rubygems/blob/master/lib/rubygems/core_ext/kernel_gem.rb#L11-L12. Do we still need to call `gem`?

Reply:
👍

Reply:
> I am not sure if I understand the comment here 🤔

You're rescuing the gem missing error, and logging that the gem doesn't exist. That's not helpful in instances where the developer thinks they have the right gem but doesn't. It is far easier to miss. IMO the app should crash with a clear error message because it requires intervention from the developer to add the right gem.

Reply:
done and done